src/serialization/SerializationService.ts

```diff
@@ -0,0 +1,11 @@
+export class SerializationService {
+    toData(object: any) : Buffer {
```

Comment:
I added `Data` type. I also realized that `BaseProxy` should work with `Data` rather than `Buffer` and fixed it too.